### PR TITLE
Wiring up aggregator and contract

### LINKED ISSUE #487: Wire Up Aggregator & Contract
Make the Aggregator Server configurable, reading contract address and blockchain connection info from config.

This includes testing that all of our subscription stuff works end-to-end.

### PR DESCRIPTION
**Pull request by willmeister**
## Description
- Wires up Mock-Aggregator with contract, reading config from  an environment file
- Fixes bugs with end-to-end functionality
- Verifies that it works!!

### Fixes
- Fixes #487 

## Contributing Agreement

- [x] I have read and understood the [Plasma Group Contributing Guide and Code of Conduct](https://github.com/plasma-group/pigi/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: packages/contracts/.local.env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DEPLOY_MNEMONIC='social spell miracle box cream brisk dash edit badge citizen tail modify'
```
Comment (ben-chain) [inline packages/contracts/.local.env]: +1 for high quality mnemonic, we are definitely building a social spell miracle box

File: packages/example-rollup/src/mock-aggregator.ts [hunk 2]
```diff
@@ -84,18 +113,53 @@ async function runAggregator() {
...
-    new DefaultSignatureProvider(Wallet.fromMnemonic(AGGREGATOR_MNEMONIC))
+    new DefaultSignatureProvider(Wallet.fromMnemonic(AGGREGATOR_MNEMONIC)),
+    DefaultSignatureVerifier.instance(),
+    transitionsPerBlock
   )
 
-  // TODO: sync blocks and remove this line
-  await aggregator.onSyncCompleted()
+  if (mockMode) {
+    // TODO: sync blocks and remove this line
```
Comment (ben-chain) [inline packages/example-rollup/src/mock-aggregator.ts]: This can be removed now, right?

File: packages/contracts/.local.env (hunk 1 above)
Comment (willmeister) [reply packages/contracts/.local.env]: OH NO! Everybody knows my ganache key!

but for real... this should have been removed by the .gitignore. I'll manually remove it.

File: packages/example-rollup/src/mock-aggregator.ts (hunk 2 above)
Comment (willmeister) [reply packages/example-rollup/src/mock-aggregator.ts]: Haha, yes! Thanks 😄

**Pull request merged**